src/features/sponsor-dashboard/components/Submissions/Modals/MultiActionModal.tsx

```
@@ -0,0 +1,249 @@
+    try {
+      switch (actionType) {
+        case 'reject':
+          rejectSubmissions.mutate(submissionIds);
+          setSelectedSubmissionIds(new Set());
+          break;
+        case 'bonus':
+          const bonusSubmissions = submissionIds.map((s) => ({
+            id: s,
+            isWinner: true,
+            winnerPosition: BONUS_REWARD_POSITION,
+          }));
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)